policyengine_us/parameters/gov/states/ar/tax/income/rates/main.yaml

```diff
@@ -0,0 +1,49 @@
+description: Arkansas taxes personal taxable income at these rates.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Arkansas income tax rate
+  reference:
+  - title: Tax Legislation for 2022
+    href: https://www.arkansashouse.org/news/post/15098/tax-legislation-for-2022#:~:text=In%20the%20most%20recent%20special,rate%20is%20reduced%20to%205.5%25.
+  - title: State of Arkansas Estimated Tax Declaration Vouchers and Instructions for Tax Year 2023
```

Comment:
don't need the 2023 estimation

Reply:
Please add the legislative reference - the legal code of Arkansas

Reply:
https://casetext.com/statute/arkansas-code-of-1987/title-26-taxation/subtitle-5-state-taxes/chapter-51-income-taxes/subchapter-2-imposition-of-tax/section-26-51-201-effective-90-days-after-sine-die-adjournment-individuals-trusts-and-estates